### ISSUE TITLE
The links list feature does not show the accessible description 

### ISSUE THREAD

**Issue by mraccess77** [FreedomScientific/VFO-standards-support#374]
The links list feature in JAWS does not show the accessible description that is provided by aria-describedby or the title attribute when an accessible name is present.  This information is important to users.   Screen reader users demand this information be available in the accessible name instead -- but showing the accessible description would allow users to access it while using compliant code such as aria-describedby or title attribute.

### Expected result
1 Go to a page with links that have accessible descriptions.
2 press JAWSKey+f7
3 Accessible names and descriptions of links are shown in the list of links.

> What did you expect to happen?

### Actual result
1 Go to a page with links that have accessible descriptions.
2 press JAWSKey+f7
3 Only accessible names of links are shown in the list of links.

#### JAWS version and build number
JAWS 2020
 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome

**Comment by JAWS-test** [comment]
Since the description can be very extensive, I don't think it makes sense that it is always shown in the link list. It should be possible to configure in the JAWS Setting Manager when descriptions are displayed. 
* This applies not only to links, but to all items for which a list can be displayed (such as form fields)
* Furthermore, this does not only apply to the lists. You should also be able to set whether descriptions are output when reading with the virtual cursor or not

**Comment by kurtjwolf** [comment]
Confirmed this issue and is part of internal bug #105760.
